.evergreen/config.yml

```diff
@@ -72,13 +72,15 @@ functions:
            fi
 
            export MONGO_ORCHESTRATION_HOME="$DRIVERS_TOOLS/.evergreen/orchestration"
+           export PROJECT_ORCHESTRATION_HOME="$PROJECT_DIRECTORY/.evergreen/orchestration"
```

Comment:
Does our `config.yml` inherit the copy in drivers-evergreen-tools, or is that version just a base that gets copied to create the config for a new driver? If we evaluate this in addition to the drivers-evergreen-tools file, I notice this is already defined there: https://github.com/mongodb-labs/drivers-evergreen-tools/blob/92dc624d1c712fa02283f14f675997b89ceb8ef2/.evergreen/config.yml#L58

Reply:
There's no inheritance for evergreen.yml, so any changes made in drivers-evergreen-tools have to be duplicated in all repositories that are based on drivers-evergreen-tools